src/shared/util.js

```diff
@@ -789,6 +789,10 @@ function stringToPDFString(str) {
   return strBuf.join("");
 }
 
+function escapeString(str) {
```

Comment:
Why is `escapeString` necessary in this PR? It's not trivial to read, so if we really need it it would need a unit test like the other `util.js` functions.